--- Source/WebCore/platform/network/create-http-header-name-table.orig	2020-11-24 14:11:54.102671419 +1000
+++ Source/WebCore/platform/network/create-http-header-name-table	2020-11-24 14:12:15.150863937 +1000
@@ -54,7 +54,7 @@
 
 http_header_names.sort()
 
-gperf_file = open('HTTPHeaderNames.gperf', 'w')
+gperf_file = open('HTTPHeaderNames.gperf', 'w', newline='\n')
 gperf_file.write('''
 %{
 /*
--- ./Source/WebCore/css/makeSelectorPseudoElementsMap.py.orig	2020-11-25 08:42:01.215641505 +1000
+++ ./Source/WebCore/css/makeSelectorPseudoElementsMap.py	2020-11-25 08:42:15.715862161 +1000
@@ -57,7 +57,7 @@
 def expand_ifdef_condition(condition):
     return condition.replace('(', '_').replace(')', '')
 
-output_file = open('SelectorPseudoElementTypeMap.gperf', 'w')
+output_file = open('SelectorPseudoElementTypeMap.gperf', 'w', newline='\n')
 
 output_file.write("""
 %{
--- ./Source/WebCore/css/makeSelectorPseudoClassAndCompatibilityElementMap.py.orig	2020-11-25 10:56:11.973008355 +1000
+++ ./Source/WebCore/css/makeSelectorPseudoClassAndCompatibilityElementMap.py	2020-11-25 10:56:39.109387662 +1000
@@ -57,7 +57,7 @@
 def expand_ifdef_condition(condition):
     return condition.replace('(', '_').replace(')', '')
 
-output_file = open('SelectorPseudoClassAndCompatibilityElementMap.gperf', 'w')
+output_file = open('SelectorPseudoClassAndCompatibilityElementMap.gperf', 'w', newline='\n')
 
 output_file.write("""
 %{
