--- Source/cmake/OptionsQt.cmake.orig	2020-11-23 10:40:19.859260136 +1000
+++ Source/cmake/OptionsQt.cmake	2020-11-23 11:47:24.807239044 +1000
@@ -722,9 +722,9 @@
     endif ()
 endif ()
 
-if (APPLE)
-    SET_AND_EXPOSE_TO_BUILD(HAVE_QOS_CLASSES 1)
-endif ()
+#if (APPLE)
+#    SET_AND_EXPOSE_TO_BUILD(HAVE_QOS_CLASSES 1)
+#endif ()
 
 if (ENABLE_MATHML)
     SET_AND_EXPOSE_TO_BUILD(ENABLE_OPENTYPE_MATH 1)
