--- ./Source/JavaScriptCore/assembler/ARM64Assembler.h.orig	2022-12-30 16:08:07.402011556 +1000
+++ ./Source/JavaScriptCore/assembler/ARM64Assembler.h	2022-12-30 16:08:30.189748031 +1000
@@ -2664,7 +2664,7 @@
 
     static void cacheFlush(void* code, size_t size)
     {
-#if OS(IOS)
+#if OS(DARWIN)
         sys_cache_control(kCacheFunctionPrepareForExecution, code, size);
 #elif OS(LINUX)
         size_t page = pageSize();
